bin/commands/kuzzle-start.js

```diff
@@ -79,8 +79,10 @@ module.exports = function () {
          ████████████████████████████████████`);
         firstAdmin.check()
           .then((res) => {
-            console.log(notice('[ℹ] There is actually no administrator user. You can use the CLI or the back-office to create one.'));
-            console.log(notice('[ℹ] Entering no-administrator mode: everyone has administrator rights.'));
+            if (res.result.total === 0) {
+              console.log(notice('[ℹ] There is actually no administrator user. You can use the CLI or the back-office to create one.'));
```

Comment:
actually is a false friend in French. 
=> `There is no administrator user set yet`? or just `No administrator user found`?